### DISCUSSION TITLE
sillytavern with google colab

### DISCUSSION THREAD

**Discussion by Mertsudo** [erew123/alltalk_tts#336] [Q&A] [answered]
Kabold working fine but sillytavern refuses to work installed your suggested extension too no luck need help thanks in advance.

**Comment by erew123** [comment, marked as answer]
Hi @Mertsudo 

I made some additional notes about pressing the Enter button when changing the IP/URL, perhaps that is your issue? https://github.com/erew123/alltalk_tts/blob/alltalkbeta/system/SillyTavern%20Extension/For%20AllTalk%20V2/Instructions.txt

Thanks

**Reply by Mertsudo** [reply]
changing alltalk.js settings with notepad localhost to api link fixed problem ıt was not saving from the beginning link ı put really weird bug thanks again

**Reply by Mertsudo** [reply]
I found another bug checking Only narrate "quotes" and Ignore *text, even "quotes", inside asterisks* not working too. Reloading makes unchecked.

**Reply by erew123** [reply]
@Mertsudo Those are part of SillyTavern and not AllTalk, so I have no overview of how they work. 

Irrespective, if you are enabling the Narrator function, and wish to have a silent narrator (things inside Asterisks) you can do that with
 
![image](https://github.com/user-attachments/assets/8263f13e-b0dd-4035-bd47-9b28b148fea2)
